src/components/traces/details/timeline/tagsTable.jsx

```diff
@@ -17,7 +17,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import _ from 'lodash';
-import validUrl from '../../../../utils/validUrl';
+import config from '../../../../utils/tagsConfig';
 
```

Comment:
tagsConfig is too generic of a name, how about something like TagValuesTransformer?